### PR TITLE
Categories update

### PR DESCRIPTION
**Pull request by babenek**
Categories were updated to use them in ML training
Other and Private Key are not used in ML dataset

### CODE REVIEW

File: meta/9128de73.csv [hunk 1]
```diff
@@ -39,11 +39,11 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 23340,f192be78,GitHub,9128de73,data/9128de73/src/f192be78.go,134:134,F,F,,,F,F,,,,,0,0,F,F,F,Generic Secret
 25923,7e5488c0,GitHub,9128de73,data/9128de73/src/7e5488c0.go,41:41,F,F,,,F,F,,,,,0,0,F,F,F,Generic Secret
 26100,524e5f37,GitHub,9128de73,data/9128de73/src/524e5f37.go,75:75,F,F,,,F,F,,,,,0,0,F,F,F,Generic Secret
-30522,cc3efc56,GitHub,9128de73,data/9128de73/test/cc3efc56.yaml,7:7,Template,T,6,1858,F,F,Any,,,Unknown,5.49,1852,F,F,F,Other
-30523,cc3efc56,GitHub,9128de73,data/9128de73/test/cc3efc56.yaml,16:16,Template,T,6,1958,F,F,Any,,,Unknown,5.51,1952,F,F,F,Other
-30536,cc3efc56,GitHub,9128de73,data/9128de73/test/cc3efc56.yaml,8:8,Template,T,4,1736,F,F,Any,,,Unknown,5.53,1732,F,F,F,Other
+30522,cc3efc56,GitHub,9128de73,data/9128de73/test/cc3efc56.yaml,7:7,T,T,6,1858,F,F,Any,,,Unknown,5.49,1852,F,F,F,Generic Secret
+30523,cc3efc56,GitHub,9128de73,data/9128de73/test/cc3efc56.yaml,16:16,T,T,6,1958,F,F,Any,,,Unknown,5.51,1952,F,F,F,Generic Secret
+30536,cc3efc56,GitHub,9128de73,data/9128de73/test/cc3efc56.yaml,8:8,T,T,4,1736,F,F,Any,,,Unknown,5.53,1732,F,F,F,Other
```
Comment (babenek) [inline meta/9128de73.csv:44]:
```suggestion
30536,cc3efc56,GitHub,9128de73,data/9128de73/test/cc3efc56.yaml,8:8,T,T,4,,F,F,Any,,,Unknown,5.53,1732,F,F,F,Other
```

File: meta/9128de73.csv [hunk 2]
```diff
@@ -39,11 +39,11 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 23340,f192be78,GitHub,9128de73,data/9128de73/src/f192be78.go,134:134,F,F,,,F,F,,,,,0,0,F,F,F,Generic Secret
 25923,7e5488c0,GitHub,9128de73,data/9128de73/src/7e5488c0.go,41:41,F,F,,,F,F,,,,,0,0,F,F,F,Generic Secret
 26100,524e5f37,GitHub,9128de73,data/9128de73/src/524e5f37.go,75:75,F,F,,,F,F,,,,,0,0,F,F,F,Generic Secret
-30522,cc3efc56,GitHub,9128de73,data/9128de73/test/cc3efc56.yaml,7:7,Template,T,6,1858,F,F,Any,,,Unknown,5.49,1852,F,F,F,Other
-30523,cc3efc56,GitHub,9128de73,data/9128de73/test/cc3efc56.yaml,16:16,Template,T,6,1958,F,F,Any,,,Unknown,5.51,1952,F,F,F,Other
-30536,cc3efc56,GitHub,9128de73,data/9128de73/test/cc3efc56.yaml,8:8,Template,T,4,1736,F,F,Any,,,Unknown,5.53,1732,F,F,F,Other
+30522,cc3efc56,GitHub,9128de73,data/9128de73/test/cc3efc56.yaml,7:7,T,T,6,1858,F,F,Any,,,Unknown,5.49,1852,F,F,F,Generic Secret
```
Comment (babenek) [inline meta/9128de73.csv:42]:
```suggestion
30522,cc3efc56,GitHub,9128de73,data/9128de73/test/cc3efc56.yaml,7:7,T,T,6,,F,F,Any,,,Unknown,5.49,1852,F,F,F,Generic Secret
```

File: meta/9128de73.csv [hunk 3]
```diff
@@ -39,11 +39,11 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
...
+30523,cc3efc56,GitHub,9128de73,data/9128de73/test/cc3efc56.yaml,16:16,T,T,6,1958,F,F,Any,,,Unknown,5.51,1952,F,F,F,Generic Secret
```
Comment (babenek) [inline meta/9128de73.csv:43]:
```suggestion
30523,cc3efc56,GitHub,9128de73,data/9128de73/test/cc3efc56.yaml,16:16,T,T,6,,F,F,Any,,,Unknown,5.51,1952,F,F,F,Generic Secret
```

**Review by babenek — COMMENTED** [review]
skip cert obfuscation

**Review by xDizzix — APPROVED** [review]

**Review by iuriimet — APPROVED** [review]

**Pull request merged**